app_scraping/templates/app_scraping/index.html

```diff
@@ -0,0 +1,22 @@
+{% if resort_list %}
+    <ul>
+    {% for resort in resort_list %}
+        <li>{{ resort.resort_name }}</li>
+        <p>Trails Open: {{ resort.trails_open }}/{{ resort.total_trails }}</p>
+        <p>Lifts Open: {{ resort.lifts_open }}/{{ resort.total_lifts }}</p>
+        <p>Acres Open: {{ resort.acres_open }}</p>
+        <p>Terrain Open: {{ resort.terrain_percent }}%</p>
+    {% endfor %}
+    </ul>
+{% else %}
+    <p>No resorts available</p>
+{% endif %}
+
```

Comment:
Only one empty line at the end of the files. The rest of the whitespace is unnecessary.

Reply:
Done